core/dbt/adapters/base/column.py

```diff
@@ -60,6 +60,7 @@ def is_float(self):
             "float",
             "double precision",
             "float8",
+            "double",
```

Comment:
Is there a way to test this change?

Reply:
I might be mistaken but I think those existing tests (based on the types) are valid for a postgres database, postgres doesn't have a `double` type, it's an alias in snowflake and a pure type in databaricks, so the tests would need to run on one of those to be valid?

Reply:
That tracks. @jtcohen6, do you know of any way to help test this? The change itself is simple enough though.